### ISSUE TITLE
When I opened the app in iPad reveal showing only 1/3 rd of the screen how can I fit the 75% of the screen  can you please help me ?

### ISSUE THREAD

**Issue by saikishoreagrapalem** [John-Lluch/SWRevealViewController#783]
Actually I found the one issue that I had recognize initDefaultProperties method there will be splitting _rearViewRevealWidth,_rightViewRevealWidth we are setting here but how can I fix this issues when I open in iPad it should cover 75 of the screen please help me on this ??????

**Comment by iDevelopper** [comment]
self.revealViewController().rearViewRevealWidth = UIScreen.main.bounds.size.width * 0.75

**Comment by saikishoreagrapalem** [comment]
yes its working fine now then I need to give the blur view for that reamanining 25 of the revealed screen in both iPhone and iPad can you please help me ?
